IOHomework/src/main/java/ro/sciit/iohomework/StatisticsExporter.java

```diff
@@ -0,0 +1,59 @@
+package ro.sciit.iohomework;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+public class StatisticsExporter {
+
+    /**
+     * @param salesStatistic provides results exported in a file
+     * @throws IOException
+     */
+    public void writeStatsToFile(SalesStatistics salesStatistic) throws IOException {
+
+        File f = new File("E:\\CursJava\\sci\\IOHomework\\salesStatistics.txt");
+        if (f.exists()) {
+            f.delete();
+        } else {
+            f.createNewFile();
+        }
+
+        String nl = System.getProperty("line.separator");
+
+        try (BufferedWriter bw = new BufferedWriter(new FileWriter(f))) {
+            List<Product> products = salesStatistic.getProductStatistics(salesStatistic.getProductsHashMap());
```

Comment:
This is bad design: salesStatistic.getProductStatistics(salesStatistic.getProductsHashMap())
You should just call salesStatistic.getProductStatistics();